chapter3/exercise1.js

```diff
@@ -0,0 +1,30 @@
+(function() {
+
+   var min = function(firstNumber, secondNumber) {
+        var result;
+
+        if((typeof firstNumber == "undefined") || (typeof secondNumber == "undefined")) {
+            result = "You don't pass two needed arguments to function.";
+            return console.log(result);
+            
+        } else if(((typeof firstNumber == "number") || (firstNumber instanceof Number)) && 
+        ((typeof secondNumber == "number") || (secondNumber instanceof Number))) {
+            if(firstNumber < secondNumber) {
+                result = firstNumber;
+                return console.log(result);
+            } else if (secondNumber < firstNumber) {
+                result = secondNumber;
+                return console.log(result);
```

Comment:
jezeli najpiew w warunkach okreslisz result a pozniej zrobisz returna to bedziesz mial jeden punkt wyjscia zamiast kilku